toolset/setup/linux/frameworks/ulib.sh

```diff
@@ -28,6 +27,17 @@ if [ $? -ne 0 ]; then
    sudo apt-get install -y libcap2-bin
 fi
 
+# We need to install mongo-c-driver (we don't have a ubuntu package)
+RETCODE=$(fw_exists ${IROOT}/mongo-c-driver.installed)
+if [ "$RETCODE" != 0 ]; then
```

Comment:
I noticed that other frameworks (ffead-cpp) must do the same installation of mongo-c-driver,
I thought that this would spare duplication of work...